packages/mobile/src/services/offline-downloader/offline-storage.ts

```diff
@@ -142,8 +139,8 @@ export const getLocalTrackCoverArtDestination = (
   return path.join(getLocalTrackDir(trackId), getArtFileNameFromUri(uri))
 }
 
-export const getLocalTrackCoverArtPath = (trackId: string, size: string) => {
-  return path.join(getLocalTrackDir(trackId), `${size}.jpg`)
+export const getLocalTrackCoverArtPath = (trackId: string) => {
+  return path.join(getLocalTrackDir(trackId), '1000x1000.jpg')
```

Comment:
should we make this from the size enum or something? It seems pretty unlikely to change though.

Reply:
at least a const in this file would be nice actually